app/main.py

```diff
@@ -2,0 +1,5 @@
+def copy_file(command: str) -> None:
+    list_files = command.split()
```

Comment:
Consider using more descriptive variable names instead of `list_files`. For example, `command_parts` would be more descriptive and align with the checklist's recommendation for variable naming.